cardstack/src/components/SwitchSelector/SwitchSelector.tsx

```diff
@@ -1,226 +1,92 @@
 
   const toggleItem = useCallback(
-    (index: number, callOnPress = true) => {
-      if (options.length <= 1 || index === null || isNaN(index)) return;
-      animate(index / options.length, selected / options.length);
-
-      if (callOnPress && onPress) {
-        onPress(options[index].value);
-      }
-
+    (index: number) => () => {
       setSelected(index);
+      onPress?.(options[index]);
     },
-    [animate, onPress, options, selected]
-  );
-
-  const responderEnd = useCallback(
-    (_evt: GestureResponderEvent, gestureState: PanResponderGestureState) => {
-      if (disabled) return;
-      const swipeDirection = getSwipeDirection(gestureState);
-
-      if (
-        swipeDirection === SWIPE_DIRECTION.RIGHT &&
-        selected < options.length - 1
-      ) {
-        toggleItem(selected + 1);
-      } else if (swipeDirection === SWIPE_DIRECTION.LEFT && selected > 0) {
-        toggleItem(selected - 1);
-      }
-    },
-    [disabled, options.length, selected, toggleItem]
-  );
-
-  const panResponder = useMemo(
-    () =>
-      PanResponder.create({
-        onStartShouldSetPanResponder: shouldSetResponder,
-        onMoveShouldSetPanResponder: shouldSetResponder,
-        onPanResponderRelease: responderEnd,
-        onPanResponderTerminate: responderEnd,
-      }),
-    [responderEnd]
+    [onPress, options]
   );
 
-  useEffect(() => {
-    if (value !== undefined) {
-      toggleItem(options.findIndex(option => option.value === value));
-    }
-  }, [options, toggleItem, value]);
-
-  const optionsMap = useMemo(
+  const renderOptions = useMemo(
     () =>
-      options.map((element, index) => {
+      options.map((option, index) => {
         const isSelected = selected === index;
 
         return (
           <Touchable
             key={index}
-            disabled={disabled}
             flex={1}
-            flexDirection="row"
             justifyContent="center"
-            alignItems="center"
-            onPress={() => toggleItem(index)}
-            height={height}
-            borderRadius={borderRadius}
+            onPress={toggleItem(index)}
+            style={buttonStyle}
+            height="95%"
+            backgroundColor={isSelected ? 'teal' : 'transparent'}
           >
             <Text
               fontWeight="600"
               fontSize={13}
               textAlign="center"
               color={isSelected ? 'black' : 'white'}
             >
-              {element.label}
+              {option.label}
             </Text>
           </Touchable>
         );
       }),
-    [borderRadius, disabled, height, options, selected, toggleItem]
+    [buttonStyle, options, selected, toggleItem]
   );
 
   return (
     <Container
+      height={height}
+      borderWidth={1.5}
       flexDirection="row"
-      accessibilityLabel={accessibilityLabel}
-      testID={testID}
+      borderColor="whiteLightOpacity"
       backgroundColor="darkGrayOpacity"
+      alignItems="center"
```

Comment:
<img width="300" alt="Screen Shot 2022-03-08 at 7 10 03 PM" src="https://user-images.githubusercontent.com/16714648/157233737-c690a459-1d71-4d52-97ab-ad0cbb04791d.png">
<img width="300" alt="Screen Shot 2022-03-08 at 7 09 52 PM" src="https://user-images.githubusercontent.com/16714648/157233759-833d00fd-3075-4565-a613-78cf451c35f2.png">

As you can see screenshots, item Touchable overlaps container border, so suggest this change.

```suggestion
      alignItems="center"
      style={{
        paddingHorizontal: 1,
      }}
```